views/accountSettings.html

```diff
@@ -79,6 +79,9 @@
 						</div>
 					</div>
 
+				{{#if showPAssertedIdentity}}
+          
+          
```

Comment:
Is there any reason for the extra whitelines?

Reply:
Mmmm, looks like a bad conflict resolution.

I actually merged 4.3 into this branch, which I don't think I should have done until the `master` PR was merged.

Let me try to fix that (I'll close those PRs and try reverting that last merge).